src/game/objects/bridge.h

```diff
@@ -2,6 +2,7 @@
 
 #include "global/types.h"
 
+bool IsSameBridgeSector(int32_t x, int32_t y, int32_t z, const ITEM_INFO *item);
```

Comment:
I believe we can make this function static within `bridge.c`, as it seems to be unused in other modules?

Reply:
Yep. Done.